lms/envs/dev.py

```diff
@@ -49,6 +49,22 @@
                             dev_env=True,
                             debug=True)
 
+# cdodge: This is the specifier for the MongoDB (using GridFS) backed static content store
```

Comment:
I'd like for the codebase to move away from including people's names/initials unless there's a good reason for it.